README.md

```diff
@@ -8,7 +8,7 @@ An example app using the Stytch Javascript SDK with HTML and Express.js
 2. Install dependencies: `npm install`
 3. Create your own .env file by running `cp .env_template .env`
 4. Fill in the placeholder values for `STYTCH_PROJECT_ID` and `STYTCH_SECRET` in your `.env` file. Get your Stytch API credentials from the API keys tab in your [Stytch dashboard](https://stytch.com/dashboard/api-keys).
-5. Fill in the placeholder value for `STYTCH_PUBLIC_TOKEN` in `public/signupOrLogin.html`. This can also be found on your [Stytch dashboard](https://stytch.com/dashboard).
+5. Fill in the placeholder value for `STYTCH_PUBLIC_TOKEN` in `public/signupOrLogin.html`. This can also be found on your [Stytch dashboard](https://stytch.com/dashboard/api-keys).
```

Comment:
I like that the public token is part of the `.env` setup now! I think these lines should be merged into one item.